sandbox/grommet-app/src/pages/layouts/kinds/NavigationalSidebar/NavigationalSidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Box, Page, PageContent, PageHeader } from "grommet"
+import { Previous } from "grommet-icons";
+import { Link } from "react-router-dom";
+import { NavSidebar, RoutedAnchor } from "../../../../components";
+import { navItems } from "./navItems";
+
+const defaultSelected = navItems["Dashboard"];
+
+export const NavigationalSidebar = () => {
+  return (
+    <Box direction="row" fill>
+      <NavSidebar
+        title="Private Cloud"
+        active={defaultSelected}
+        items={navItems}
+        flex="grow"
+      />
+      <Page pad={{ bottom: "xlarge" }}>
+        <PageContent alignSelf="start">
+          <PageHeader
+            title="Navigational Sidebar"
```

Comment:
```suggestion
            title="Navigational sidebar"
```
sentence casing since it's not a product name.

Reply:
done.